service/test/Makefile.mk

```diff
@@ -0,0 +1,373 @@
+                          test/DifferenceSignalTest.cpp \
+                          test/DomainControlTest.cpp \
+                          test/ExceptionTest.cpp \
+                          test/geopm_test.cpp \
+                          test/geopm_test.hpp \
+                          test/geopm_test_helper.cpp \
+                          test/HelperTest.cpp \
+                          test/IOGroupTest.cpp \
+                          test/LevelZeroAcceleratorTopoTest.cpp \
+                          test/LevelZeroDevicePoolTest.cpp \
+                          test/LevelZeroIOGroupTest.cpp \
+                          test/MSRIOGroupTest.cpp \
+                          test/MSRIOTest.cpp \
+                          test/MSRFieldControlTest.cpp \
+                          test/MSRFieldSignalTest.cpp \
+                          test/MockAcceleratorTopo.hpp \
+                          test/MockControl.hpp \
+                          test/MockIOGroup.hpp \
+                          test/MockMSRIO.hpp \
+                          test/MockNVMLDevicePool.hpp \
+                          test/MockLevelZeroDevicePool.hpp \
+                          test/MockLevelZero.hpp \
+                          test/MockPlatformIO.hpp \
+                          test/MockPlatformTopo.cpp \
+                          test/MockPlatformTopo.hpp \
+                          test/MockSDBus.hpp \
+                          test/MockSDBusMessage.hpp \
+                          test/MockServiceProxy.hpp \
+                          test/MockSignal.hpp \
+                          test/MockSSTIO.hpp \
+                          test/MockSSTIoctl.hpp \
+                          test/NVMLAcceleratorTopoTest.cpp \
+                          test/NVMLIOGroupTest.cpp \
+                          test/PlatformIOTest.cpp \
+                          test/PlatformTopoTest.cpp \
+                          test/RawMSRSignalTest.cpp \
+                          test/SharedMemoryTest.cpp \
+                          test/SSTControlTest.cpp \
+                          test/SSTIOGroupTest.cpp \
+                          test/SSTSignalTest.cpp \
+                          test/SSTIOTest.cpp \
+                          test/TimeIOGroupTest.cpp \
+                          # end
+
+test_geopm_test_LDADD = libgeopmd.la \
+                        libgmock.a \
+                        libgtest.a \
+                        # end
+
+test_geopm_test_CPPFLAGS = $(AM_CPPFLAGS) -Iplugin
+test_geopm_test_CFLAGS = $(AM_CFLAGS)
+test_geopm_test_CXXFLAGS = $(AM_CXXFLAGS)
+
+# Target for building test programs.
+gtest-checkprogs: $(GTEST_TESTS)
+
+PHONY_TARGETS += gtest-checkprogs
+
+$(GTEST_TESTS): test/gtest_links/%:
+	mkdir -p test/gtest_links
+	rm -f $@
+	ln -s $(abs_srcdir)/test/geopm_test.sh $@
+
+coverage: check
+	lcov --no-external --capture --directory src --output-file coverage-service.info
+	genhtml coverage-service.info --output-directory coverage-base
```

Comment:
```suggestion
	genhtml coverage-service.info --output-directory coverage-service
```

Reply:
The coverage target needs to be updated in the base dir/test Makefile.mk as well.  It still has refs to the service directory.